technical-reports/format/groups.md

```diff
@@ -228,15 +228,15 @@ For example:
 
 </aside>
 
-### GUI tools
+### Design tools
```

Comment:
Same comment as above. I'd prefer to keep this as "GUI tools" as this advice can apply to tools that are not design tools but still have a GUI.